lib/vehicle_factory.rb

```diff
@@ -0,0 +1,16 @@
+class VehicleFactory
+
+  def initialize
+  end
+
+  def create_vehicles(api_data)
+    vehicles = api_data.map do |data|
+    Vehicle.new(
+      vin: data[:vin_1_10], 
+      make: data[:make], 
+      model: data[:model], 
+      year: data[:model_year], 
+      engine: :ev)
```

Comment:
Breaking up the arguments for a call to `.new` onto separate lines is a great way to keep lines short and make everything readable. Be sure to match the closing paren to the line with the opening paren:
```
Vehicle.new(
  vin: data[:vin_1_10], 
  make: data[:make], 
  model: data[:model], 
  year: data[:model_year], 
  engine: :ev
)
```